Civi/Api4/Service/Spec/Provider/ProductAttributeSpecProvider.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+namespace Civi\Api4\Service\Spec\Provider;
+
+use Civi\Api4\Service\Spec\FieldSpec;
+use Civi\Api4\Service\Spec\RequestSpec;
+
+/**
+ * @service
+ * @internal
+ */
+class ProductAttributeSpecProvider extends \Civi\Core\Service\AutoService implements Generic\SpecProviderInterface {
+
+  public function modifySpec(RequestSpec $spec): void {
+    // @todo FIXME how to name this correctly?
+    $field = new FieldSpec('options2', $spec->getEntity(), 'Array');
```

Comment:
You could modify the existing field instead of replacing it:

```suggestion
    $field = $spec->getFieldByName('options');
```